lib/puppet/agent/locker.rb

```diff
@@ -23,13 +23,13 @@ def lock
         lockfile.unlock
       end
     else
-      fail Puppet::LockError, 'Failed to acquire lock'
+      fail Puppet::LockError, _('Failed to acquire lock')
     end
   end
 
   # @deprecated
   def running?
-    Puppet.deprecation_warning <<-ENDHEREDOC
+    Puppet.deprecation_warning _(<<-ENDHEREDOC)
```

Comment:
Yes. I thought that was pretty cool. You can also call methods on a `HEREDOC` anchor; it turns out Ruby treats the anchor itself as a String object.